tests/compas/geometry/test_distance.py

```diff
@@ -0,0 +1,10 @@
+from compas.geometry import Point
+from compas.geometry import Line
+from compas.geometry import closest_point_on_segment_xy
+
+
+def test_closest_point_segment_xy():
```

Comment:
would you mind including non-boundary cases as well?

Reply:
There you have it, I parametrized the test and added a non boundary case.